pkg/manifests/cache.go

```diff
@@ -101,8 +104,8 @@ func (c *ManifestCache) Expire(id string) {
 	c.cache.Remove(id)
 }
 
-func (l *cacheLine) live(dur time.Duration) bool {
-	return l.created.After(time.Now().Add(-dur))
+func (l *cacheLine) live(expiry, jitter time.Duration) bool {
+	return l.created.After(time.Now().Add(-expiry).Add(-time.Duration(rand.Int63n(int64(jitter)))))
```

Comment:
the jitter should be a stable for ech line, we should add it to the go struct, and set it to a random value between 0 and 10m on create (then when it's refreshed, regenerated the jitter).

This will recompute it each time the line is checked for liveness, which isn't ideal.

Reply:
It's done now. PTAL.